JsonLedgerApiSample/src/main/java/com/example/store/IntegrationStore.java

```diff
@@ -24,6 +28,170 @@ public class IntegrationStore {
     private String sourceSynchronizerId = null;
     private String lastIngestedRecordTime = null;
     private final String treasuryParty;
+    private final HashMap<String, Balances> userBalances = new HashMap<>();
+
+    public BigDecimal getBalance(InstrumentId instrumentId, String depositId) {
+        return userBalances.getOrDefault(depositId, new Balances()).getBalance(instrumentId);
+    }
+
+    private record TransferInfo(
+            String sender,
+            String depositId) {
+    }
+
+    private class TransactionParser {
+        static final String SPLICE_PREFIX = "splice.lfdecentralizedtrust.org/";
```

Comment:
Do we consider these strings to be defined by splice or by the token standard? Depending on the answer, I would suggest to move these constants to `models/TokenStandard.java` or `models/Splice.java`.

Reply:
token standard. will move them there.